test/AWS.jl

```diff
@@ -381,7 +415,7 @@ end
 
 @testset "json" begin
     @testset "secrets_manager" begin
-        secret_name = "AWS.jl-Test-Secret" * lowercase(Dates.format(now(Dates.UTC), "yyyymmddTHHMMSSZ"))
+        secret_name = "AWS.jl-Test-Secret" * lowercase(Dates.format(now(Dates.UTC), "yyyymmddTHHMMSSsssZ"))
```

Comment:
Just a best practise:
```suggestion
        secret_name = "AWS.jl-Test-Secret" * lowercase(Dates.format(now(Dates.UTC), dateformat"yyyymmdd\THHMMSSsssZ"))
```